employee/sealed.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Starting from a `read_only` struct, this program
+ * restores full privileges using a previous `sealed`
+ * capabilty.
+ */
+
+#include "../include/common.h"
+#include <assert.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/sysctl.h>
+
+
+
+int main()
+{
+	// Request special capability `sealcap` from the operating system
+	// in order to use it as key to seal `new_small_salary`
+	void *__capability sealcap;
+	size_t sealcap_size = sizeof(sealcap);
+	if (sysctlbyname("security.cheri.sealcap", &sealcap, &sealcap_size, NULL, 0) < 0){
+		error("Fatal error. Cannot get `securiy.cheri.sealcap`.");
+		exit(1);
+	}
+	assert(cheri_perms_get(sealcap) & CHERI_PERM_SEAL);
+	uint8_t *new_small_salary;
+	// Seal `new_small_salary` using previously requested `sealcap`
+	new_small_salary = (uint8_t *) malloc(sizeof(uint8_t));
+	new_small_salary =  (uint8_t *) cheri_seal(new_small_salary, sealcap);
```

Comment:
Double space after `=`.

Reply:
I believe `clang_format` would have fixed it automatically :) I haven't run it yet, though.

Reply:
Ah. We should always run formatters before pushing new commits, because this sort of thing distracts reviewers otherwise.

Reply:
Yes, you are right. I have a pre-commit hook in my TODO list...

Reply:
Fixed in 68148ac.

Reply:
I've never been sure how reliable pre commits are so I run things manually myself. But if a pre commit works for you, that's fine!